util/environmentCheck.js

```diff
@@ -0,0 +1,9 @@
+const check = function check(variable) {
```

Comment:
Tests are missing for this function. You can also think of putting in the isValidNode() and InValidNodeException in this module itself.